modules/govuk/manifests/apps/content_audit_tool.pp

```diff
@@ -1,165 +0,0 @@
-# == Class: govuk::apps::content_audit_tool
-#
-# App details at: https://github.com/alphagov/content-audit-tool
-#
-# === Parameters
-#
-#
-# [*db_hostname*]
-#   The hostname of the database server to use in the DATABASE_URL.
-#   Default: undef
-#
-# [*db_port*]
-#   The port of the database server to use in the DATABASE_URL.
-#   Default: undef
-#
-# [*db_allow_prepared_statements*]
-#   The ?prepared_statements= parameter to use in the DATABASE_URL.
-#   Default: undef
-#
-# [*db_name*]
-#   The database name to use in the DATABASE_URL.
-#
-# [*db_password*]
-#   The password for the database.
-#   Default: undef
-#
-# [*db_username*]
-#   The username to use in the DATABASE_URL.
-#
-# [*google_analytics_govuk_view_id*]
-#   The view id of GOV.UK in Google Analytics
-#   Default: undef
-#
-# [*google_client_email*]
-#   Google authentication email
-#   Default: undef
-#
-# [*google_private_key*]
-#   Google authentication private key
-#   Default: undef
-#
-# [*oauth_id*]
-#   Sets the OAuth ID for using GDS-SSO
-#   Default: undef
-#
-# [*oauth_secret*]
-#   Sets the OAuth Secret Key for using GDS-SSO
-#   Default: undef
-#
-# [*port*]
-#   The port that it is served on.
-#   Default: 3213
-#
-# [*publishing_api_bearer_token*]
-#   The bearer token to use when communicating with Publishing API.
-#   Default: undef
-#
-# [*redis_host*]
-#   Redis host for Sidekiq.
-#   Default: undef
-#
-# [*redis_port*]
-#   Redis port for Sidekiq.
-#   Default: undef
-#
-# [*secret_key_base*]
-#   The key for Rails to use when signing/encrypting sessions.
-#
-# [*sentry_dsn*]
-#   The URL used by Sentry to report exceptions
-#
-class govuk::apps::content_audit_tool(
-  $db_hostname = undef,
-  $db_port = undef,
-  $db_allow_prepared_statements = undef,
-  $db_name = 'content_audit_tool_production',
-  $db_password = undef,
-  $db_username = 'content_audit_tool',
-  $enable_procfile_worker = true,
-  $google_analytics_govuk_view_id = undef,
-  $google_client_email = undef,
-  $google_private_key = undef,
-  $oauth_id = undef,
-  $oauth_secret = undef,
-  $port = '3213',
-  $publishing_api_bearer_token = undef,
-  $redis_host = undef,
-  $redis_port = undef,
-  $secret_key_base = undef,
-  $sentry_dsn = undef,
-) {
-  $app_name = 'content-audit-tool'
-
-  govuk::app { $app_name:
-    app_type          => 'rack',
-    port              => $port,
-    sentry_dsn        => $sentry_dsn,
-    health_check_path => '/',
-    asset_pipeline    => true,
-  }
```

Comment:
I'm guessing that there's at least one resource here that won't be cleaned up by just removing the configuration here.